packages/components/switch/README.md

```diff
@@ -0,0 +1,25 @@
+# @consolelabs/switch
+
+The Switch component is used as an alternative between checked and not checked
+states.
+
+Please refer to the
+[documentation](https://web-design-system-consolelabs.vercel.app/?path=/docs/ui-switch--docs)
+for more information.
```

Comment:
For further details, visit our [Switch Documentation](https://web-design-system-consolelabs.vercel.app/?path=/docs/ui-switch--docs).